modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -492,8 +492,11 @@ enum GcalContinuum {
   # GcalContinuum IrGreyBodyHigh
   IR_GREY_BODY_HIGH
 
-  # GcalContinuum QuartzHalogen
-  QUARTZ_HALOGEN
+  # GcalContinuum QuartzHalogen 5W
+  QUARTZ_HALOGEN5
+
+  # GcalContinuum QuartzHalogen 100W
+  QUARTZ_HALOGEN100
```

Comment:
Will this update break Explore?  There are two Quartz Halogen lamps so we need to distinguish them.

Reply:
I think we're OK since the enum value has been added to lucuma-core. If there is a problem lucuma-schemas and explore probably need to be updated to the new lucuma-core.